routes/new-project.js

```diff
@@ -5,21 +5,16 @@ var utils = require("./utils");
 module.exports = function(config) {
   return function(req, res) {
     var qs;
-    var projectName = req.params.projectName;
-    if(!projectName) {
-      res.status(400).send({error: "No project name specified"});
-      return;
-    }
-
     var cur = req.query.now || (new Date()).toISOString();
     var project = {
-      title: projectName,
+      title: config.constants.NEW_PROJECT,
```

Comment:
Yeah you're right! I think I'm actually gonna make this an env variable (so that its consistent on the front-end + backend) and create a separate file for constants